### PR TITLE
[10.0]search invoice lines on the commercial partner

### PR DESCRIPTION
**Pull request by chafique-delli**
To correct issue: https://github.com/Eficent/stock-rma/issues/113

### CODE REVIEW

File: rma_account/wizards/rma_add_invoice.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
                 </group>
                 <separator string="Select Invoices lines to add"/>
                 <field name="invoice_line_ids"
-                       domain="[('invoice_id.partner_id', '=', partner_id),
+                       domain="[('invoice_id.commercial_partner_id', '=', partner_id),
```
Comment (AaronHForgeFlow) [inline rma_account/wizards/rma_add_invoice.xml]: Thanks for this Pull Request!

Perhaps here we can try here with:

`domain="[('invoice_id.commercial_partner_id', 'child_of', partner_id),`

That way if will take both, the commercial_partner, the partner and all the children. Do you agree?

Comment (chafique-delli) [reply rma_account/wizards/rma_add_invoice.xml]: Ok, I do that.

Comment (AaronHForgeFlow) [reply rma_account/wizards/rma_add_invoice.xml]: Thank you. I will test if that woks before merging. Does it work on your side?

Comment (chafique-delli) [reply rma_account/wizards/rma_add_invoice.xml]: Yes, it works on my side

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: LGTM

**Pull request merged**